src/js/pages/Service.js

```diff
@@ -58,6 +52,9 @@ class Service extends Component {
     }
 
     axios
+      .create({
+        headers: {'Accept-Language': lang }
```

Comment:
Your end `}` has a space before it but not after the leading `{`

I'm sorry to even point this out, but I saw it so...

Reply:
stop trying to crush my individuality

Reply:
Enjoy yourself spidey!

![image](https://user-images.githubusercontent.com/272675/34747481-165a0dfe-f55e-11e7-9e7a-0e9b7f3264de.png)